src/installer.ts

```diff
@@ -148,6 +151,11 @@ export async function checkKamelNeedsUpdate(versionToUse? : string): Promise<boo
 			}
 			versionToUse = latestversion.result.trim();
 		}
+		if (runtimeVersionSetting && runtimeVersionSetting.toLowerCase() !== versionToUse.toLowerCase()) {
+			versionToUse = runtimeVersionSetting;
```

Comment:
I think the the failing test is reporting a real problem. The version to use is modified here from "bogusversion" to "1.0.0-RC2", then the method declares that there is no kamel update needed.

Reply:
You are correct - this is what launched me into looking at why I can't run tests locally (see https://github.com/camel-tooling/vscode-camelk/pull/335). Can't debug a test if I can't run ANY tests.